src/scripts/main.js

```diff
@@ -22,7 +22,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
-}
+  if (Object.keys(data).length === 0) {
+    return false;
+  } else {
```

Comment:
There is no need to use 'if...else' as code will not go further after 'return' anyway